packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.types.ts

```diff
@@ -238,6 +238,12 @@ export interface IDocumentCardActivityPerson {
    */
   initials?: string;
 
+  /**
+   * Whether initials are calculated for phone numbers.
+   * @defaultvalue false
+   */
+  calculateInitialsForPhoneNumber?: boolean;
```

Comment:
that's oddly specific to phone numbers. Would it be more straightforward to call it `allowNumberInitials`?

Also it might be a good idea to describe in the comments what special exception scenarios might be appropriate for allowing this. (Projects which start with a number, Personas which represent a phone number...)

Reply:
The code logic itself uses the concept of phone numbers, supporting the "ext" and "x" extension syntax in US numbers in the regular expression. The property can only consist of numbers, except for the optional "x" or "ext" at the end, after all special characters have been removed. 

So without this change, "1234" will be seen as a phone number with empty initials. "1234A" will not be seen as a phone number, with initials "1", With this change, and setting calculateInitialsForPhoneNumber to true, "1234" would also get initials "1".

I agree that calculateInitialsForPhoneNumber is very specific. But allowNumberInitials doesn't quite capture the logic either, as without this fix, a contact with name "1 2A" would still get initials "12", so number only initials is already supported.

What we want to support is that values representing what we believe are phone numbers (only numbers with an optional "x" or "ext" at the end followed by more numbers) can either get empty initials or normal initials. The use case is values representing project names which are incorrectly interpreted as phone numbers (and given no initials).

I'm in no way married to the calculateInitialsForPhoneNumber name :-)

Reply:
ok gotcha. What about `allowPhoneInitials` ? A little shorter?

Reply:
Renamed. Also added more context to the comments. Ok to use "numbers", or should I use the more correct term "numerals"?